File: source/user-application/app/src/main/java/vn/com/user_application/core/models/Response.java
package vn.com.user_application.core.models;

public class Response<T> {
    private T data;
    private String message;
    private int status = 1;

    public Response() {}

    public Response(T data, String message, int status) {
        this.data = data;
        this.message = message;
        this.status = status;
    }

    public int getStatus(){
        return this.status;
    }

    public T getData() {
        return data;
    }


    public String getMessage() {
        return message;
    }

    @Override
    public String toString() {
        return "Response{" +
                "data=" + data +
                ", message='" + message + '\'' +
                ", status=" + status +
                '}';
    }
}

File: source/manager-application/Models/Response.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace manager_application.Models
{
    public class Response<T>
    {
        [JsonProperty(PropertyName = "data")]
        public T data { get; set; }
        [JsonProperty(PropertyName = "message")]
        public string Message { get; set; }
        [JsonProperty(PropertyName = "status")]
        public int Status { get; set; }

        Response() { }
    }
}
